src/pages/messages/components/NviCandidatesList.tsx

```diff
@@ -44,25 +45,42 @@ export const NviCandidatesList = () => {
         {t('tasks.nvi.nvi_control')}
       </Typography>
 
-      <Box
-        sx={{
-          mb: '1rem',
-          mx: { xs: '0.5rem', md: 0 },
-          display: 'flex',
-          flexDirection: 'column',
-          gap: '1rem',
-        }}>
-        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '1rem' }}>
-          <NviStatusFilter sx={{ flex: '1 13rem' }} />
+      <Grid container columns={16} spacing="1rem" sx={{ px: { xs: '0.5rem', md: 0 }, mb: '1rem' }}>
```

Comment:
Ingen særlig fan av MUI Grid, men siden målet nå er at Søket for NVI skal oppleves likt som det for Oppgaver tenker jeg det er like greit å bare basere seg på samme oppsett som der også. Så har derfor tatt utgangspunkt i samme Grid layout som for oppgaver her